db/models/student.js

```diff
@@ -14,7 +14,7 @@ module.exports = db.define('student', {
   },
   imageUrl: Sequelize.STRING
 }, {
-  defaultScope: {
+  defaultScope: { //pretty cool use of defaultScope
```

Comment:
I like it.  Just be aware that including associated models all the time is not always a good thing.  It can be a huge performance hit (doing all these joins all the time, even if you may not need the associated campus model, is expensive!).

Additionally, it kind of makes your students routes a little less restful.  Now when you make a get request to students route, you are getting additional info that you may not need.  This may not be a problem now, but imagine making a project where all your routes are returning all this extra attached data.  It gets messy, and other devs looking at your code could get confused... you should be able to know from a glance at a route, what it is exactly you are getting.  Student get request should give you student instance, campus get request should give you campus instance... etc.  With includes data, maybe have a custom route for that?

This is more of a philosophical thing, and I haven't been able to explain this as well as I would like.  But if you have more questions, feel free to ask me!  (or any instructors)